Sources/SwifterSwift/SceneKit/SCNVector3Extensions.swift

```diff
@@ -139,19 +139,6 @@ public extension SCNVector3 {
     static func / (vector: SCNVector3, scalar: SceneKitFloat) -> SCNVector3 {
         return SCNVector3(vector.x / scalar, vector.y / scalar, vector.z / scalar)
     }
-
-    /// SwifterSwift: Divide self with a scalar
-    ///
-    ///     SCNVector3(10, 20, -30) /= 3 -> SCNVector3(3/10, 0.15, -30)
-    ///
-    /// - Parameters:
-    ///   - vector: self.
-    ///   - scalar: scalar value.
-    /// - Returns: result of division of the given CGPoint with the scalar.
-    static func /= (vector: inout SCNVector3, scalar: SceneKitFloat) {
-        // swiftlint:disable:next shorthand_operator
```

Comment:
@thisIsTheFoxe no I meant just remove the `// swiftlint:disable` since you didn't need it anymore